src/corona_hakab_model/main.py

```diff
@@ -48,18 +69,24 @@ def main():
     else:
         matrix_consts = MatrixConsts()
 
-    gm = GenerationManger(circles_consts=circles_consts, matrix_consts=matrix_consts)
+    if args.matrix_data and args.population_data:
+        matrix_data = MatrixData.import_matrix_data(args.matrix_data)
+        population_data = PopulationData.import_population_data(args.population_data)
 
-    if args.sub_command == "generate":
-        with open(args.output, "wb") as w:
-            write(gm.matrix_data.matrix, w)
-        return
+    else:
+        gm = GenerationManger(circles_consts=circles_consts, matrix_consts=matrix_consts)
+        if args.sub_command == 'generate':
+            gm.matrix_data.export('matrix')
```

Comment:
this is exported upon generation by default anyway, no?
i think you would export the matrix twice this way.

Reply:
No enabled by default, but I should still refactor it.